test/blackbox-tests/test-cases/melange/public.t/my_project/dune

```diff
@@ -1,5 +1,4 @@
 (melange.emit
- (package pkg)
```

Comment:
We discussed this privately, but leaving here a note for posterity: 

> The approach I ended up taking is to always put js artifacts of public libs in node_modules, regardless which package or dune project they belong to:
> - If two public libraries belong to the same package, relative paths like `require('../foo/bar.js')` will work because the original source layout is honored under the node_modules folder
> - If two public libs belong to different packages, non-relative paths like `require('foo/bar.js')` will work because at the top level there will be a node_modules folder that includes foo.

> This approach breaks only in one case: when the entries in melange.emit belong to the same package as some public library. I was able to fix this by removing the package entry from melange.emit, so the entries in emit become "private" (in dune's `Lib_info.Status.t` terms).

Rudi answered:

> For executables, we have a `package` field. But this is mainly useful when you want to:
> - Install the executable
> - Include the executable as part of the test suite for the package. so that `dune build -p pkg @runtest` will see this executable

We probably want to fix this at some point in the future, either by removing `package` field in `melange.emit` (at least in the short term), or figuring out how to perform installation of emit entries.